### PR TITLE
Add git author

### LINKED ISSUE #575: upload_nightly.sh: git error
See: http://build.servo.org/builders/mac-nightly/builds/200/steps/shell__2/logs/stdio

I requires a valid email address, and `servo@servo-mac2.(none)` is not.

I see 3 options:

- set the domain name of the machine then it doesn't return `(none)`
- set an email address under `common/init.sls`. Would it work to just add a `email` field under `servo:user.present:`?
- user `--author` when commiting

/cc @aneeshusa

### PR DESCRIPTION
**Pull request by paulrouget**
fix https://github.com/servo/saltfs/issues/575

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/14935)

### CODE REVIEW

File: etc/ci/upload_nightly.sh [hunk 1]
```diff
@@ -52,7 +52,8 @@ update_brew() {
        > "${tmp_dir}/Formula/servo-bin.rb"
 
   git -C "${tmp_dir}" add ./Formula/servo-bin.rb
-  git -C "${tmp_dir}" commit -m "Version bump: ${version}"
+  git -C "${tmp_dir}" commit --author="Tom Servo <servo@servo.org>" \
```
Comment (aneeshusa) [inline etc/ci/upload_nightly.sh]: nit: move `--author=...` to next line

File: etc/ci/upload_nightly.sh [hunk 2]
```diff
@@ -52,7 +52,8 @@ update_brew() {
...
+      -m "Version bump: ${version}"
```
Comment (aneeshusa) [inline etc/ci/upload_nightly.sh]: nit: use full form (`--message=...`) for consistency

**Review by aneeshusa — CHANGES_REQUESTED** [review]
LGTM aside from 2 nits.

**Comment by paulrouget** [comment]
Comments addressed.

**Review by aneeshusa — APPROVED** [review]

**Comment by aneeshusa** [comment]
Thanks for the fix! I'm actually not sure how this was working before...
@bors-servo r+

**Pull request merged**